packages/cli/lib/commands/transform.ts

```diff
@@ -16,12 +17,29 @@ interface RunOptions extends Command {
   project: string;
   variable: Map<string, string>;
   enableBufferMonitor: boolean;
+  issuer?: string;
+  clientId: string;
+  clientSecret: string;
+  authParam: Map<string, string>;
 }
 
 export default function (pipelineId: NamedNode, basePath: string, log: Debugger) {
   return async function (command: RunOptions) {
     const { from, to, project, debug, variable, enableBufferMonitor } = command
 
+    log.enabled = debug
+
+    if (command.issuer) {
+      const authConfig: AuthConfig = {
```

Comment:
I assumed here that if the `--issuer` flag is set, other flags are as well. Not sure if there is an easy way to do this with `commander`